.travis.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Xaptum, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License
+
+sudo: required
+
+language: c
+
+services:
+  - docker
+
+compiler: gcc
```

Comment:
Let's remove the `compiler: gcc` and `language: c` lines. Just because we're not using them, so it might be confusing